.github/workflows/docs-build.yml

```
@@ -79,3 +79,68 @@ jobs:
         if: steps.check-skip.outputs.skip-build != 'true'
         run: |
           poe docs-build
+
+  vercel-preview:
+    name: Vercel Preview
+    needs: detect-changes
+    if: >
+      github.event.pull_request.head.repo.full_name == github.repository
+    # && needs.detect-changes.outputs.changed == 'true'
```

TODO: 

- [ ] Uncomment this before merging. (Restricts builds to only when there are relevant changes.)

```suggestion
    # && needs.detect-changes.outputs.changed == 'true'
```